src/test/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/integration/wiremock/ExternalUsersApiMockServer.kt

```diff
@@ -1015,6 +1055,27 @@ class ExternalUsersApiMockServer : WireMockServer(WIREMOCK_PORT) {
     )
   }
 
+  fun stubNotFound(url: String) {
```

Comment:
prob rename to stubGetNotFound - I think some of the PUTs return it too

Reply:
Done